src/pages/landing/landing.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {useState} from 'react';
+import {Header} from "../../others/components/Header";
+import {Content} from "../../others/components/Content";
+import {Spacer} from "../../others/components/Spacer";
+import {Text} from "../../others/components/Text";
+import {Button} from "../../others/components/Button";
+import {useTranslation} from "react-i18next";
+import {Input} from "../../others/components/Input";
+import PhoneInput from "../../others/components/PhoneInput";
+import styles from "./landing.module.css";
+import { ImgLocationPin } from '../../medias/images/UGT_Asset_UI_LocationPin';
+import {useNavigate} from "react-router-dom";
+import ToggleButton from "../../others/components/ToggleButton";
+import {useSosInfoContext} from "../../others/contexts/sosInfo";
+
+interface EmergenciesOptions {
+    selectedItem: string;
+    onClick: (e: string) => void;
+}
+
+const EmergenciesOptions = ({selectedItem, onClick}: EmergenciesOptions) => {
+    const emergency_codes = ["1", "2", "3"];
+    const { t } = useTranslation();
+
+    return <div style={{display: "flex", justifyContent: "space-between", flexWrap: "wrap"}}>
+        {emergency_codes.map(e => <ToggleButton key={e}
+                                                active={e === selectedItem}
+                                                value={t(`landing_emergency_${e}`)}
+                                                onClick={() => onClick(e)} />)}
+    </div>
+}
+
+const Landing = () => {
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+
+    const { currentValue, updateValue } = useSosInfoContext();
+
+    const onSubmit = () => {
+        console.log('Alerted:', currentValue);
```

Comment:
Hi Olya, not pushing to local storage yet since I wasn't sure about the use case of recovering from local storage again. 

Here would be best place to have it.